### PR TITLE
Update project core definition

### PR DESCRIPTION
**Pull request by ishitasati11**
Updated with the requested changes

### CODE REVIEW

**Review by KrutikaGaonkar — COMMENTED** [review]
reviewed, as you noticed the links are not applicable/useful in most cases. Can we remove the ontology link property?

File: DataModel/Project/Core/Project-core-definition.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+Project-core:
+  description: Describes the core details of the project like project ID, summary,
+    overall design, publication related details and any other supplementary links
+    associated with the project
+  properties:
+    properties:
```
[the remark this answers is not in the document]
Comment (ishitasati11) [reply DataModel/Project/Core/Project-core-definition.yaml:6]: Removed

File: DataModel/Project/Core/Project-core-definition.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+      project_id:
+        description: Primary key or unique identifier for a project
+        type: string
+        pattern: "^DO-[0-9]{4}$"
+        example: P034
```
[the remark this answers is not in the document]
Comment (ishitasati11) [reply DataModel/Project/Core/Project-core-definition.yaml:11]: Updated

**Comment by ishitasati11** [comment]
Removed ontologies in most places, updated the files and pushed to #12 @KrutikaGaonkar

**Pull request closed without merging**